ci-operator/step-registry/ipi/conf/gcp/public-dns-zone/ipi-conf-gcp-public-dns-zone-commands.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+if ! test -f "${SHARED_DIR}/xpn.json"; then
+  echo "'${SHARED_DIR}/xpn.json' not found, abort." && exit 1
+fi
+
+GCP_BASE_DOMAIN="$(< ${CLUSTER_PROFILE_DIR}/public_hosted_zone)"
+
+echo "Reading variables from ${SHARED_DIR}/xpn.json..."
+HOST_PROJECT=$(jq -r '.hostProject' "${SHARED_DIR}/xpn.json")
+
+if [[ -z "${BASE_DOMAIN}" ]]; then
+  BASE_DOMAIN="$(jq -r '.baseDomain' "${SHARED_DIR}/xpn.json")"
```

Comment:
When users is using public domain from host project, `BASE_DOMAIN` should be always be overridden accordingly, no need the `if` condition, right?